src/js/components/Skeleton/stories/TableSkeleton.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import { Box, DataTable } from 'grommet';
+import { columns, DATA } from '../../DataTable/stories/data';
+
+export const TableSkeleton = () => (
+  <Box align="center" pad="large" skeleton>
+    <DataTable columns={columns} data={DATA} step={10} />
```

Comment:
I hadn't done table yet since I don't think it's a useful example so far. In a real usage you wouldn't have data yet and we wouldn't want the headers to skeletonize I think. To do this correctly I think requires some changes to DataTable

Reply:
That makes sense, I can go back and remove this story